imaging/models.py

```diff
@@ -46,9 +49,11 @@ class TwoPhotonImaging(BaseExperimentalData):
                                  help_text="in nm")
     recording_wavelength = models.FloatField(null=True, blank=True,
                                  help_text="in nm. Can be array for multispectral imaging. TODO: deal with arrays?")
+								 # does django have a way of encoding small arrays in regular tables? I guess not if they are variable size... so it would need to be json
```

Comment:
An Array is OK - i don't think anyone is going to search on this, it is just there for completeness.

Is this standard sql though?